instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Comments from './Comments';
+
+
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments
+        };
+    }
+    render() {
+        return (
+            <div>
+                {this.state.comments.map((c, i) => <Comments key = {i} comment ={c} />)}
```

Comment:
Thnks about using more verbose parameter names in your map such as comment , index or if you don't want it to coincide with your prop then you could prepend an a to your arguments as in aComment, aIndex to denote that it is the argument variables and not the properties